templates/zerver/register.html

```diff
@@ -118,8 +118,26 @@ <h1>{{ _('Create your account') }}</h1>
                 <div class="input-box no-validation">
                     <input type='hidden' name='key' value='{{ key }}' />
                     <input type='hidden' name='timezone' id='timezone'/>
+                    <input type="hidden" name="email_address_visibility" id="email_address_visibility"/>
                     <label for="id_email" class="inline-block label-title">{{ _('Email') }}</label>
                     <div id="id_email">{{ email }}</div>
+                    {% if not creating_new_team %}
```

Comment:
We should probably rename this to `creating_new_realm` to improve readability at some point.

Reply:
Will do this as a follow-up.